src/test/java/products/ProductsTests.java

```diff
@@ -0,0 +1,14 @@
+package products;
+
+import base.BaseTest;
+import org.testng.annotations.Test;
+import salesforce.ui.pages.ProductPage;
+import static configfile.Configuration.dotenv;
+
+public class ProductsTests extends BaseTest {
+
+//    @Test
+    public void testCreateProductWithJustRequiredFields() {
+        ProductPage productPage = loginPage.loginSuccessfully(dotenv.get("USER_NAME_SALESFORCE"), dotenv.get("PASSWORD"));
```

Comment:
The credential values could be retrieved in an Enum so Strings are not used here.

Reply:
Thanks, I have already fixed that